share/templates.d/99-generic/live/live-install.tmpl

```diff
@@ -18,8 +18,9 @@
     installpkg shim-ia32 grub2-efi-ia32-cdboot
 %endif
 %if basearch in ("i386", "x86_64"):
-    installpkg biosdevname syslinux
+    installpkg biosdevname
     installpkg grub2-tools grub2-tools-minimal grub2-tools-extra
+    install grub2-pc-modules
```

Comment:
Why `install` instead of `installpkg`?

Reply:
Because it's a typo :) I haven't tested building a live iso yet. Thanks for catching that.